docs/video-player.md

```diff
@@ -0,0 +1,128 @@
+
+## Example
+
+For a fully working example take a look at our [examples directory](https://github.com/datocms/react-datocms/tree/master/examples).
+
+```js
+import React from 'react';
+import { VideoPlayer } from 'react-datocms';
+
+const Page = ({ data }) => (
+  <div>
+    <h1>{data.blogPost.title}</h1>
+    <VideoPlayer data={data.blogPost.cover.video} />
+  </div>
+);
+
+const query = gql`
+  query {
+    blogPost {
+      title
+      cover {
+        video {
+          # required: this field identifies the video to be played
+          muxPlaybackId
+
+          # all the other fields are not required but:
+
+          # if provided, title is displayed in the upper left corner of the video
+          title
+
+          # if provided, width and height are used to define the aspect ratio of the
+          # player, so to avoid layout jumps during the rendering.
+          width
+          height
+
+          # if provided, it shows a blurred placeholder for the video
+          blurUpThumb
+
+          # you can include more data here: they will be ignored by the component
+        }
+      }
+    }
+  }
+`;
+
+export default withQuery(query)(Page);
+```
+
+## Props
+
+The `<VideoPlayer />` components supports all the properties made available for
+`<MuxPlayer />` component from `@mux/mux-player-react` package plus `data`, which is meant to
+receive data directly in the shape they are provided by DatoCMS GraphQL API.
+
+`<Video Player />` uses the `data` prop to generate a set of props for the inner
+`<MuxPlayer />`. On this topic, also see the "Advanced usage" section below.
+
+| prop | type           | required           | description                                                      | default |
+| ---- | -------------- | ------------------ | ---------------------------------------------------------------- | ------- |
+| data | `Video` object | :white_check_mark: | The actual response you get from a DatoCMS `video` GraphQL query |         |
+
+Compared to the `<MuxPlayer />`, some prop default values are different on `<VideoPlayer />`
+
+- `disableCookies` is normally true, unless you explicitly set the prop to `false`
+- the video height and width, when available in the `data` props, are used to set `{ aspectRatio: "[width] / [height]"}` for the `<MuxPlayer />`'s `style`
+
+All the other props are forwarded to the `<MuxPlayer />` component that is used internally.
+
+## Advanced usage: the `useVideoPlayer` hook
+
+Despite we try our best to make the `<VideoPlayer />` suitable and easy to use for most normal use cases, there are situation where you may need to leverage the `<MuxPlayer />` directly. If that's the case, fill free to use the hook we provide: `useVideoPlayer`.
+
+`useVideoPlayer` takes data coming in the shape they are produced from DatoCMS API and return props that you can pass to dhe `<MuxPlayer />` component: that's pretty much what the `<VideoPlayer />` does internally.
```

Comment:
"the" instead of "dhe"

Reply:
And maybe a period or semicolon instead of the colon (:), like

...pass to the MuxPlayer component. That's pretty much what the...